apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-docs.component.ts

```diff
@@ -117,6 +163,24 @@ export class PlatformDynamicPageDocsComponent {
             code: dynamicPageFlexibleColumnExampleTsCode,
             fileName: 'platform-dynamic-page-flexible-column-example',
             component: 'PlatformDynamicPageFlexibleColumnExampleComponent'
+        },
+        {
+            language: 'typescript',
+            name: 'platform-dynamic-page-page-overflow.service.ts',
+            code: platformDynamicPagePageOverflowServiceTs,
+            fileName: 'platform-dynamic-page-page-overflow',
+            component: 'PlatformDynamicPagePageOverflowService',
+            service: true
         }
     ];
+
+    ngOnInit(): void {
+        this._overflowHandlingService.isExampleOpened.subscribe((isExampleOpened) => {
```

Comment:
would be good to unsubscribe on destroy event

Reply:
done